charts/prometheus/templates/prometheus-config-configmap.yaml

```diff
@@ -68,7 +71,9 @@ data:
           - source_labels: [__meta_kubernetes_namespace, __meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
             action: keep
             regex: default;kubernetes;https
-
+      {{- end }}
+      # TODO add a proper config flag option for disabling core-dns scraping instead of using useClusterRoles
```

Comment:
When does this TODO need to be done?[ We probably need a ticket for this.](url)

Reply:
we are currently going to give one flag option to disable clusterrole altogether